src/actions/project-text-search.js

```diff
@@ -34,6 +34,10 @@ export function clearSearchResults() {
   };
 }
 
+export function closeProjectSearch() {
+  return { type: "CLOSE_PROJECT_SEARCH" };
```

Comment:
this isn't dispatching... and the only place a dispatch is used is in the test, not in the click handler. any ideas about what is going on here?

Reply:
@codehag It is used in the click handler [here](https://github.com/princiya/debugger.html/blob/b17a22194cca393d9888fdc1bf2315ed8dc1f7c6/src/components/ProjectSearch/TextSearch.js#L191)

Reply:
but there is no dispatching that is going on, it is just calling the action which is returning an object, or maybe i am missing a wrapper somewhere that does the dispatch?

Reply:
or i just forgot how action creators work 😅

Reply:
@codehag I guess the latter one! :)